docs/readme/body.rst

```diff
@@ -554,6 +557,46 @@ Environment Variables
     ``AWSCLI_LOGIN_ROOT`` is set to ``/tmp`` then the plugin will
     look for configuration files in (``/tmp/.aws-login/``).
 
+Keyrings with WSL and Windows Credential Store
+=====================
+
+If running under WSL, you may wish to store credentials in the Windows
+Credential Store in the Windoes host operating system and access them
+fromwithin WSL. To do this, first install Python on the Windows
+host operating system if not already installeed. You can do this from
+the Microsoft Store or using Winget, or you can download the installer
+directly from the Python official site. The Python keyring module
+will need to beinstalled on the Windows Python instance. From a
+Windows command prompt, run:
+
+    pip install keyring
+
+Next, inside WSL, install the keyring-pybridge module:
+
+    pip install keyring-pybridge
+
+Still in WSL, Set the PYTHON_KEYRING_BACKEND environment variable to
+tell Python to use the pybridge keyring backend. This is shell
+specific but might look like:
+
+    export PYTHON_KEYRING_BACKEND=keyring_pybridge.PyBridgeKeyring
+
+Finally, set the KEYRING_PROPERTY_PYTHON envorinment variableto point
+to the Windows Python executable. If you need to find the full path
+to this executable, run this from the Windows command prompt:
+
+    where python.exe
+
+Inside WSL, set this environment variable like this. If Python is at
+c:\path\to\python.exe, translate the path like this:
+
+    export KEYRING_PROPERTY_PYTHON=/mnt/c/paty/to/python.exe
```

Comment:
```suggestion
    export KEYRING_PROPERTY_PYTHON='/mnt/c/paty/to/python.exe'
```

My path contained a space, which will be common for certain Windows versions.

Reply:
> My path contained a space, which will be common for certain Windows versions.

Good catch, thank you. I've implemented that change and also corrected the typo of /mnt/c/paty to /mnt/c/path.